### DISCUSSION TITLE
Allow for Closed Sidewalks without Lane order

### DISCUSSION THREAD

**Discussion by Charlie-Henry** [usdot-jpo-ode/jpo-wzdx#372] [Q&A] [answered]
Looks like the only place to supply sidewalk closures in in the `lanes` [details](https://github.com/usdot-jpo-ode/wzdx/blob/main/spec-content/objects/Lane.md). We are wondering if it is possible to report a sidewalk closure without knowing the order of lanes (which breaks business rule 2). Making the cross section of every road (including sidewalks/bike lanes/parking) a requirement is a pretty impractical for a City DOT. [vehicle_impact](https://github.com/usdot-jpo-ode/wzdx/blob/main/spec-content/enumerated-types/VehicleImpact.md) is also pretty car-centric and doesn't help this use case.

**Comment by j-d-b** [comment, marked as answer]
I think adding a value to the Vehicle Impact enumerated type for "sidewalk closed" is a great idea. I'm going to transfer this discussion to an issue for better tracking of that feature request.

**Reply by j-d-b** [reply]
See #373

**Comment by rdsheckler** [comment]
We have had requests from utility contractors for the ability to share work zone impacts on pedestrians as they frequently disturb sidewalks.  This requests is amplified when ADA concerns are applied.  I think that this is a subject that needs a special focus in WZDx when we start a new round of development.
